legate/core/utils.py

```diff
@@ -8,15 +8,19 @@
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# WITHOUT WARRANTIES OR CONDITIONS OF Hashable KIND, either express or implied.
```

Comment:
Search & replace was a little too eager here :-)

Reply:
Hah! Thanks for the catch 👍

Reply:
https://github.com/nv-legate/legate.core/pull/158/commits/2da2c8560bc27babaa1ec312143cc0ce5b4863ab